lib/kdump_utils.pm

```diff
@@ -227,6 +228,7 @@ sub configure_service {
     $self->wait_boot(bootloader_time => 300);
 
     select_console 'root-console';
+    ensure_serialdev_permissions;
```

Comment:
This looks like a better location. If done here I wonder if it's not a better place already within `wait_boot`. But then the main question is still open: Why do permissions not work originally after they had been set in system_prepare

Reply:
> This looks like a better location. If done here I wonder if it's not a better place already within `wait_boot`. But then the main question is still open: Why do permissions not work originally after they had been set in system_prepare

we are now tracking all the way to the install_service and check_upgraded_service. and 
now run with EXCLUDE_SERVICES=kdump http://openqa.suse.de/t4123407 PASSED
so the kdump in check_upgraded_service was the culprit.

This maybe caused after reboot, and check_upgraded_service was called before system_prepare. now I'm running jobs to verify my fix. stay tuned.
Thanks.